### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by daleel1**
nav bar
enabling floaters to register 
fixing buttons

### CODE REVIEW

**Review by MHDGhanemBalhawan — APPROVED** [review]

File: frontend/src/Components/Admin/Events/AdminEvents.js [hunk 1]
```diff
@@ -10,21 +10,26 @@ import VolunteerForm from "../Floaters/FloaterToVolunteer";
 const Adminevents = props => {
     return (
         <div className="events">
-            <h1>Events</h1>
-            <Popup
-                trigger={
-                    <button className="btn btn-outline-primary mb-2">
-                        add a new event
+            <div class="dummyDiv" />
```
Comment (Behnazz) [inline frontend/src/Components/Admin/Events/AdminEvents.js]: `className` not `class`

File: frontend/src/Components/Admin/Floaters/Floaters.js [hunk 2]
```diff
@@ -32,86 +32,62 @@ export default class Floaters extends React.Component {
...
                             <th scope="col">Edit</th>
                             <th scope="col">Delete</th>
                         </tr>
                         {this.state.mentors.map(floater => {
-                            return (
-                                <tr key={floater.floater_id}>
-                                    <th scope="row">{floater.floater_id}</th>
+                            return <tr key={floater.floater_id}>
+                                    <td />
+                                    {/* <th scope="row">{floater.floater_id}</th> */}
```
Comment (Behnazz) [inline frontend/src/Components/Admin/Floaters/Floaters.js]: Remove comments please

File: frontend/src/Components/Admin/Floaters/FloatersOfEvent.js [hunk 3]
```diff
@@ -14,8 +14,9 @@ export default class FloatersOfEvents extends React.Component {
     render() {
         return (
             <div>
-                <table className="table table-striped table-bordered">
-                    <tbody>
+                <table>
+                    <tbody class="table table-borderless">
```
Comment (Behnazz) [inline frontend/src/Components/Admin/Floaters/FloatersOfEvent.js]: ditto (`className`)

File: frontend/src/Components/Public/AddToVolunteerList.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import Message from "../Message/Message";
+// import Popup from "reactjs-popup";
+// import VolunteerForm from "../Admin/Floaters/FloaterToVolunteer";
```
Comment (Behnazz) [inline frontend/src/Components/Public/AddToVolunteerList.js]: Remove comments

File: frontend/src/Components/Public/AddToVolunteerList.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+                                        >
+                                            Submit
+                                        </button>
+                                        <a
+                                            className="btn btn-primary "
+                                            href="/events"
+                                        >
+                                            Back
+                                        </a>
+                                        {/* <Popup
```
Comment (Behnazz) [inline frontend/src/Components/Public/AddToVolunteerList.js]: Remove comments

File: frontend/src/Style/Events.css [hunk 6]
```diff
@@ -4,5 +4,32 @@
...
+.topBar {
+  float: top;
+  position: fixed;
+}
+.dummyDiv {
+  margin-top: 5em;
+}
+.headerEvents {
+  display: inline;
+  /* border-bottom: 1px solid rgb(41, 41, 43); */
```
Comment (Behnazz) [inline frontend/src/Style/Events.css]: No comments :)

**Review by Behnazz — CHANGES_REQUESTED** [review]

File: frontend/src/Components/Admin/Floaters/FloatersOfEvent.js (hunk 3 above)
Comment (daleel1) [reply frontend/src/Components/Admin/Floaters/FloatersOfEvent.js]: done

File: frontend/src/Components/Public/AddToVolunteerList.js (hunk 4 above)
Comment (daleel1) [reply frontend/src/Components/Public/AddToVolunteerList.js]: done

File: frontend/src/Components/Public/AddToVolunteerList.js (hunk 5 above)
Comment (daleel1) [reply frontend/src/Components/Public/AddToVolunteerList.js]: done

File: frontend/src/Style/Events.css (hunk 6 above)
Comment (daleel1) [reply frontend/src/Style/Events.css]: Done @Behnazz review it please

**Review by Behnazz — APPROVED** [review]

**Pull request merged**